src/main/java/mate/academy/model/MovieSession.java

```diff
@@ -0,0 +1,67 @@
+package mate.academy.model;
+
+import java.time.LocalDateTime;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "movie_sessions")
+public class MovieSession {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @ManyToOne
+    private Movie movie;
+    @ManyToOne
+    private CinemaHall cinemaHall;
+    private LocalDateTime showTime;
```

Comment:
specify column name for this field in snail_case format

Reply:
Ok, but could you please explain why. Because on the scheme we have camelCase name?

Reply:
<img width="407" alt="Screenshot 2023-02-03 at 11 26 20" src="https://user-images.githubusercontent.com/103888898/216563737-4cb1a953-71f0-497d-968e-7f095b799871.png">

Reply:
[ERROR] /Users/oleksii/Documents/GitHub/hibernate-create-movie-session-hw/src/main/java/mate/academy/model/MovieSession.java:21:27: Member name 'show_time' must match pattern '^[a-z][a-z0-9][a-zA-Z0-9]*$'. [MemberName]

Reply:
Checkstyle reject such name of variable